src/style.css

```diff
@@ -1 +1,11 @@
 /* styles go here */
+label{
+    display: block;
+   margin: 10px;
+    padding-left:30px;
+    
+    
+}
+fieldset{
```

Comment:
```suggestion
label{
  display: block;
  margin: 10px;
  padding-left: 30px;    
}

fieldset{
```
Please fix your formatting. Also, don't use tag selectors 
![image](https://user-images.githubusercontent.com/96409563/221654650-86a7fa73-d44f-4768-b413-87351b751395.png)